lib/page_indexer.rb

```diff
@@ -0,0 +1,30 @@
+# This class is resonsible for getting list of the most popular pages and placing them in a
+# file to be used by the html and visual differs
+
+require 'yaml'
+require_relative 'root.rb'
+require_relative 'format_searcher'
+require_relative 'page_searcher'
+require_relative 'app_config'
+
+class PageIndexer
+  GOVUK_PAGES_FILE = "#{ROOT_DIR}/config/govuk_pages.yml"
```

Comment:
This is a constant, but defined in other places as well. Perhaps we could make that a shared constant?